src/Altinn.Notifications.Core/Models/Notification/SmsSendOperationResult.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Text.Json;
+
+using Altinn.Notifications.Core.Enums;
+
+namespace Altinn.Notifications.Core.Models.Notification;
+
+/// <summary>
+/// A class representing a sms send operation update object
+/// </summary>    
+public class SmsSendOperationResult
+{
+    /// <summary>
+    /// The notification id
+    /// </summary>
+    public Guid NotificationId { get; set; }
+
+    /// <summary>
+    /// The reference to the delivery in sms gateway
+    /// </summary>
+    public string GatewayReference { get; set; } = string.Empty;
+
+    /// <summary>
+    /// The sms send result
+    /// </summary>
+    public SmsNotificationResultType? SendResult { get; set; }
```

Comment:
does it make sense that this is nullable? We will never update just the gateway reference as we only get a gateway reference if the sending has been accepted by LinkMobility.

Reply:
No :)